app/controllers/users_controller.rb

```diff
@@ -1,4 +1,15 @@
 class UsersController < Clearance::UsersController
+  def create
```

Comment:
Overriding Clearance's `create` method to add the token generation. [Source](https://github.com/thoughtbot/clearance/blob/master/app/controllers/clearance/users_controller.rb#L10)

Reply:
Is this necessary/standard implementation? Can we not use a hook? Or at least call `super`, instead of duplicating logic? Just asking, don't really know!

Reply:
This is the way that thoughtbot suggests doing it, based on their [blog post](https://thoughtbot.com/blog/email-confirmation-with-clearance) about e-mail confirmation, so I sticked to it. 

I don't think super would work here, since we would need to set the token after fetching the user.

Reply:
Got it! Makes sense.